main-packages-list.txt

```diff
@@ -10,14 +10,14 @@ ipxe-bootimgs
 ipxe-roms-qemu
 iscsi-initiator-utils
 mariadb-server
-openstack-ironic-api >= 1:16.1.1-0.20210215161314.71ebba5.el8
-openstack-ironic-conductor >= 1:16.1.1-0.20210215161314.71ebba5.el8
+openstack-ironic-api >= 1:16.2.1-0.20210317011256.e058a5a.el8
+openstack-ironic-conductor >= 1:16.2.1-0.20210317011256.e058a5a.el8
 parted
 psmisc
 python3-debtcollector >= 2.2.0-0.20201008171245.649189d.el8
 python3-dracclient
 python3-gunicorn
-python3-ironic-lib >= 4.4.1-0.20201218041209.aa7cfec.el8
+python3-ironic-lib >= 4.5.1-0.20210227011240.e7a35e4.el8
```

Comment:
This is rather old, I suspect we'll need another update soon.

Reply:
we chose to enable this version for the time being because it was the last one before the wallaby cut
bow  that we have the repos correctly set, we're going to increase the version to 4.6.1